src/spec.c

```diff
@@ -277,19 +244,22 @@ static void IndexSpec_TimedOutProc(RedisModuleCtx *ctx, WeakRef w_ref) {
     // the spec was already deleted, nothing to do here
     return;
   }
-#ifdef _DEBUG
-  RedisModule_Log(NULL, "notice", "Freeing index %s by timer", sp->name);
-#endif
+  RedisModule_Log(RSDummyContext, REDISMODULE_LOGLEVEL_VERBOSE, "Freeing index %s by timer", sp->name);
```

Comment:
In general this is less efficient because Redis compares strings in order to decide the log level. Not sure its critical.

Reply:
Right, but I think this is fine in this case